### ISSUE TITLE
A version command

### ISSUE THREAD

**Issue by GetPsyched** [ATLauncher/ATLauncher#914]
### The Problem

I'm the package maintainer for ATLauncher over at [nixpkgs](https://github.com/NixOS/nixpkgs). Generally, we like to add a minor version test as a sanity check if not an entire test suite. This ensures that the package is working correctly.

### The Solution

A CLI command like `atlauncher version` or `atlauncher --version` that shows the current version of the launcher.

### Have you checked this issue doesn't already exist?

- [X] I am sure this issue doesn't exist already

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Doomsdayrs** [comment]
Mistag? Also what happened to my commit?

**Comment by RyanTheAllmighty** [comment]
> Mistag? Also what happened to my commit?

Yeah I ended up adding it in master, then realised you had a PR for it so reverted it and the tag
